atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -236,10 +238,32 @@ def _make_labels(self):
 
         return labels
 
+    def get_fs_task_id(self):
+        task_id = None
+        fs_result = self.workflow.prebuild_results.get(AddFilesystemPlugin.key)
+        if fs_result is not None:
+            try:
```

Comment:
I wonder if this is easier to read:
```python
try:
    task_id = int(fs_result['filesystem-koji-task-id'])
except KeyError:
    self.log.error("expected filesystem-koji-task-id in result of %s plugin",
                   AddFilesystemPlugin.key)
except ValueError:
    self.log.exception("invalid taskID")
```

Also, there's no need to log the actual invalid task_id value. The `ValueError` exception will provide it:
```python
>>> int('asfdasfd')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: 'asfdasfd'
invalid literal for int() with base 10: 'asfdasfd'
```